source/js/buyers-guide/analytics-events.js

```diff
@@ -86,6 +86,8 @@ function getQuerySelectorEvents(pageTitle, productName) {
       category: `buyersguide`,
       action: `submit a product button tap`,
       label: `submit a product button tap on ${pageTitle}`,
+      // Custom properties (not sent to GA)
+      conditionalQuery: `.btn-recommend`,
```

Comment:
`.btn-recommend` button is currently on multiple pages on PNI (just not the product page). This is the least change I could think of to make the console error go away.

Reply:
looks good to me! we can re-evaluate if this errors still pops up, or comes back, on pages that are important in the future =)